### DISCUSSION TITLE
I18n / l10n для мобильного приложения PastVu

### DISCUSSION THREAD

**Discussion by ars-burov** [PastVu/pastvu#690] [Ideas / Идеи и предложения]
Предлагаю добавить локализацию для статических строк в мобильное приложение, чтобы можно было показывать апп людям не говорящим на русском.

**Comment by kabalin** [comment]
Идея хорошая! Приложение это проект уважаемого @pelixpng, попробуйте создать тикет в репозитарии https://github.com/pelixpng/PastVuApp

**Reply by pelixpng** [reply]
Позже добавлю i18n
